OpenEdXMobile/src/main/java/org/edx/mobile/deeplink/Screen.java

```diff
@@ -8,6 +8,9 @@
     public static final String COURSE_DASHBOARD = "course_dashboard";
     public static final String COURSE_VIDEOS = "course_videos";
     public static final String COURSE_DISCUSSION = "course_discussion";
+    public static final String COURSE_DATES = "course_dates";
+    public static final String COURSE_HANDOUTS = "course_handout";
+    public static final String COURSE_ANNOUNCEMENTS = "course_announcement";
```

Comment:
The name of the screen should also be `course_announcements` like the variable's name.